packages/jest-core/src/__tests__/TestScheduler.test.js

```diff
@@ -28,8 +29,13 @@ jest
         onTestStart() {},
       })),
     {virtual: true},
-  );
-
+  )
+  .mock('@jest/transform', () => {
```

Comment:
why are we mocking this when it returns the actual module?

Reply:
this is to allow the test (`"with runtime error: ..."`) to spy and mock the `transform.createScriptTransformer` function. Without the "`__esModule: true`", the spyOn failed: 
```
TypeError: Cannot redefine property: createScriptTransformer
        at Function.defineProperty (<anonymous>)
```